assets/js/googlesitekit/widgets/components/WidgetAreaRenderer.js

```diff
@@ -41,8 +41,9 @@ import WidgetCellWrapper from './WidgetCellWrapper';
 import { isInactiveWidgetState } from '../util/is-inactive-widget-state';
 import InViewProvider from '../../../components/InViewProvider';
 const { useSelect } = Data;
+const DEFAULT_CLASS_APPENDIX = '--boxes'
 
-export default function WidgetAreaRenderer( { slug, totalAreas } ) {
+export default function WidgetAreaRenderer( { slug = DEFAULT_CLASS_APPENDIX, totalAreas } ) {
```

Comment:
@shayannosrat I don't think the problem here is related to the `slug` - it's related to the `style` of the widget area. `boxes` is a widget area style, not a widget area slug.

We need to investigate why the `style` variable in this component can be `undefined` - it never should be.

Reply:
@felixarntz  okay thank you for the hint, will have a further look into it 🚀